lua/nvim-tree/diagnostics.lua

```diff
@@ -1,7 +1,7 @@
 local a = vim.api
 local utils = require'nvim-tree.utils'
 local view = require'nvim-tree.view'
-local get_diagnostics = vim.lsp.diagnostic.get_all
```

Comment:
That changes minimal required version of nvim supported by nvim-tree. It should be guarded I think.
```lua
local has_07 = vim.fn.has('nvim-0.7') == 1
```